app/services/report_importer.rb

```diff
@@ -95,7 +93,11 @@ def import_log_messages
     end
   end
 
-  def inspect_report
+  def report_status
+    ReportStatusCalculator.new(:counters => raw['status']).calculate
+  end
+
+  def notify_on_report_error(mail_error_state = :puppet_error_state)
```

Comment:
I wasn't sure if this is used elsewhere (e.g., plugins), though I think it is not.
Will move to a mandatory argument